components/eamxx/src/physics/mam/mam_coupling.hpp

```diff
@@ -585,14 +592,20 @@ void compute_vertical_layer_heights(const Team& team,
     "Given column index does not correspond to given team!");
 
   const auto dz = ekat::subview(dry_atm.dz, column_index);
-  auto z_iface  = ekat::subview(dry_atm.z_iface, column_index);
-  auto z_mid    = ekat::subview(dry_atm.z_mid, column_index);
+  const auto z_iface  = ekat::subview(dry_atm.z_iface, column_index);
+  const auto z_mid    = ekat::subview(dry_atm.z_mid, column_index);
+  const auto qv = ekat::subview(dry_atm.qv, column_index);
+  const auto p_mid = ekat::subview(dry_atm.p_mid, column_index);
+  const auto T_mid = ekat::subview(dry_atm.T_mid, column_index);
+  const auto pseudo_density = ekat::subview(dry_atm.p_del, column_index);
+  // NOTE: we are using dry qv. Does calculate_dz require dry or wet?
```

Comment:
Has this question been resolved? Are we sure we are using the right mixing ratio? If this has been resolved we should remove the confusing comment.

Reply:
Yes, it has. I removed this old comment.